kaos/api/schema.py

```diff
@@ -0,0 +1,32 @@
+"""Contains the schemas used to verify JSON requests/responses as well as their validators."""
+
+from jsonschema.validators import Draft7Validator
+
+SEARCH_SCHEMA = {
+    '$schema': 'https://json-schema.org/schema#',
+    'type': 'object',
+    'additionalProperties': False,
+    'properties': {
+        'Target': {
+            'type': 'array',
+            'items': {'type': "number"},
+            'minItems': 2,
+            'maxItems': 2,
+        },
+        'POI': {
+            'type': 'object',
+            'additionalProperties': False,
+            'properties': {
+                'startTime': {'type': 'string'},
+                'endTime': {'type': 'string'},
+            },
+            'required': ['startTime', 'endTime'],
+        },
+        'PlatformID': {
```

Comment:
This doesn't match our API specification:

> PlatformID: an optional N-element list of satellite IDs. If provided, only the satellites in this list will be used to perform visibility computations. If omitted, all satellites in the database will be used.

Reply:
Yeah, I miss read the API spec